README.md

```diff
@@ -1,54 +1,42 @@
-Depositors of Morpho Vault v2 earn from the underlying protocols without having to actively manage the risk of their position.
+> [!NOTE]
+> Vault v2 instances are distinguished between:
+> - **Morpho Vaults**: Vault v2 with the [Morpho registry](LINK) abdicated. Learn more about Morpho Vaults and their benefits [here](LINK).
+> - **Standard Vaults**: Vault v2 that can supply to any protocol. They don't get all the Morpho Vaults benefits. In particular, Vault v2 has been audited only in the context of the Morpho Market V1 and Morpho Vault V1 adapters.
+
+Vault v2 enables anyone to create [non-custodial](#non-custodial-guarantees) vaults that allocate assets to any protocols, including Morpho Market v1, Morpho Market v2, and Morpho Vault v1.
+Depositors of Vault v2 earn from the underlying protocols without having to actively manage their position.
 Management of deposited assets is the responsibility of a set of different roles (owner, curator and allocators).
-The active management of invested positions involves enabling and allocating liquidity to protocols.
 
-[Morpho Vault v2](./src/VaultV2.sol) is [ERC-4626](https://eips.ethereum.org/EIPS/eip-4626) and [ERC-2612](https://eips.ethereum.org/EIPS/eip-2612) compliant.
+[Vault v2](./src/VaultV2.sol) is [ERC-4626](https://eips.ethereum.org/EIPS/eip-4626) and [ERC-2612](https://eips.ethereum.org/EIPS/eip-2612) compliant.
 The [VaultV2Factory](./src/VaultV2Factory.sol) deploys instances of Vaults v2.
 All the contracts are immutable.
 
 ## Overview
 
 ### Adapters
 
-Vaults can allocate assets to arbitrary protocols and markets via adapters, or use an adapter registry to add restrictions to allowed adapters.
-The curator enables adapters to hold positions on behalf of the vault.
+Vaults can allocate assets to arbitrary protocols and markets via separate contracts called adapters.
+They hold positions on behalf of the vault.
 Adapters are also used to know how much these investments are worth (interest and loss realization).
-Because adapters hold positions in protocols where assets are allocated, they are susceptible to accrue rewards for those protocols.
-To ensure that those rewards can be retrieved, each adapter has a skim function that can be called by the vault's owner.
-
-Adapters for the following protocols are currently available:
 
-- [Morpho Market v1](./src/adapters/MorphoMarketV1Adapter.sol).
-  This adapter allocates to any Morpho Market v1, constrained by the allocation caps (see [Id system](#id-system) below).
-  The adapter holds a position on each respective market, on behalf of the vault v2.
-- [Morpho Vault v1](./src/adapters/MorphoVaultV1Adapter.sol).
-  This adapter allocates to a fixed Morpho Vault v1 (v1.0 and v1.1).
-  The adapter holds shares of the corresponding Morpho Vault v1 (v1.0 and v1.1) on behalf of the vault v2.
```

Comment:
I would keep both the mention of rewards & the currently available adapter descriptions.